Projects/Core/DesignSystem/Sources/CheckBox/DotoriCheckBox.swift

```diff
@@ -0,0 +1,126 @@
+import UIKit
+
+public final class DotoriCheckBox: UIControl {
+
+    private let checkedView: UIImageView = {
+        let view = UIImageView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.isHidden = true
+        view.image = UIImage.checkmark.withRenderingMode(.alwaysTemplate)
+        view.tintColor = .white
+        return view
+    }()
+
+    public var isChecked: Bool = false {
+        didSet {
+            updateState()
+        }
+    }
+
+    private var hitRadiusOffset: CGFloat = 10
+
+    private var checkedViewInsets: UIEdgeInsets = UIEdgeInsets(
+        top: 5,
+        left: 5,
+        bottom: 5,
+        right: 5
+    ) {
+        didSet {
+            layoutIfNeeded()
+        }
+    }
```

Comment:
```suggestion

    private var hitRadiusOffset: CGFloat = 10

    private var checkedViewInsets: UIEdgeInsets = UIEdgeInsets(
        top: 5,
        left: 5,
        bottom: 5,
        right: 5
    ) {
        didSet {
            layoutIfNeeded()
        }
    }

    public var isChecked: Bool = false {
        didSet {
            updateState()
        }
    }
```
private property를 위로 올려주세요

Reply:
수정했습니다.
https://github.com/Dotori-app/Dotori-iOS/pull/13/commits/9e9303dbc8e75cd89fcf16825828d3d6f82c74ad